infra/build/functions/base_images.py

```diff
@@ -110,6 +110,48 @@ def run_build(steps, images, tags=None, build_version=MAJOR_TAG):
                              use_build_pool=False)
 
 
+def get_arm_manifest_steps():
+  images = [f'{TAG_PREFIX}/base-builder', f'{TAG_PREFIX}/base-runner']
+  steps = []
+  for image in images:
+    steps.extend(get_push_manifest_steps(image))
+  return steps
+
+
+def get_push_manifest_steps(image):
+
+  arm_version = f'{image}-testing-arm'
```

Comment:
What's the difference between arm and arm64 in this context?

Reply:
`-testing-arm` refers to the image I built on GCE and pushed manually. I renamed them arm_testing_image and arm64_manifest_image.